group_experimental.go

```diff
@@ -274,42 +274,47 @@ func (g *Group) GetMemberCount() (uint64, error) {
 }
 
 func (g *Group) GetMemberFromIndex(index uint64) (string, string, ObjectTypeEnum, error) {
-	var curi *C.char
-	defer C.free(unsafe.Pointer(curi))
+	var curi *C.tiledb_string_t
 
-	var cname *C.char
-	defer C.free(unsafe.Pointer(cname))
+	var cname *C.tiledb_string_t
 
 	var objectTypeEnum C.tiledb_object_t
-	ret := C.tiledb_group_get_member_by_index(g.context.tiledbContext, g.group, C.uint64_t(index), &curi, &objectTypeEnum, &cname)
+	ret := C.tiledb_group_get_member_by_index_v2(g.context.tiledbContext, g.group, C.uint64_t(index), &curi, &objectTypeEnum, &cname)
 	if ret != C.TILEDB_OK {
 		return "", "", TILEDB_INVALID, fmt.Errorf("Error getting member by index for group: %s", g.context.LastError())
 	}
+	defer C.tiledb_string_free(&curi)
```

Comment:
`tiledb_string_free` returns a result, which I don't know what to do with here.

Reply:
Looks like [it always returns `TILEDB_OK`](https://github.com/TileDB-Inc/TileDB/blob/5298c2252a7ebb34021439553be84b7e5a1bf3d3/tiledb/api/c_api/string/string_api.cc#L50-L55). I think it's safe to ignore the return value; especially since `tiledb_group_get_member_by_index_v2` succeeded at this point, which is what the caller cares about.